README.md

```diff
@@ -112,7 +112,7 @@ The complete feature set includes the following.
 | Toggle the Beta badge         | `/beta`                           |
 | Get coordinates               | `/xyz`                            |
 | Toggle God mode               | `/god`                            |
-| Immune to melee and guns      | `/demigod`                        |
+| Immune to non-instakill       | `/demigod`                        |
```

Comment:
I think this description is actually more confusing; turrets and shotguns are one-hit-kill, yet demigod provides immunity for those.

Reply:
Turrets and shotguns do actual damage, they aren't one-hit kill.

Reply:
the damage they inflict in a single shot is greater than player health, therefore the average player can reasonably consider them as "instakill" even tho we coders know that they technically aren't such behind the scenes, hence why this description is confusing

Reply:
I believe the damage is computed on the body part that was shot. I believe I have survived a hit from a shotgun, and survived 2 bullets from a turret.

Reply:
I looked at the code, and indeed I was wrong. I don't ever remember surviving or seeing someone survive a gunshot of any kind, but I can see in `ShootGun` that the shotgun damage is based on distance and doesn't involve body parts, it becomes non-lethal beyond 15 meters. As for turrets, each shot inflicts a fixed 50% damage within 30 meters, and none beyond.

Reply:
Yeah, I thought so. It was either distance/body part 😄